mne/minimum_norm/inverse.py

```diff
@@ -1010,6 +1010,14 @@ def _apply_inverse(evoked, inverse_operator, lambda2, method, pick_ori,
     return (stc, residual) if return_residual else stc
 
 
+def _apply_inverse_gen(evoked, inverse_operator, lambda2, method, pick_ori,
+                       prepared, label, method_params, return_residual,
+                       use_cps):
+    yield _apply_inverse(evoked, inverse_operator, lambda2, method, pick_ori,
+                         prepared, label, method_params, return_residual,
+                         use_cps)
```

Comment:
this does not seem to be used

Reply:
That was for the averageTFR, just cruft, good catch